addons/spreadsheet/static/src/chart/plugins/odoo_chart_feature_plugin.js

```diff
@@ -103,6 +117,35 @@ export class OdooChartFeaturePlugin extends OdooUIPlugin {
         });
     }
 
+    getAvailableChartRegions(chartId) {
+        const definition = this.getters.getChartDefinition(chartId);
+        if (definition.type !== "odoo_geo" && definition.type !== "geo") {
+            return [];
+        }
+        if (!definition.region || definition.region === "world") {
+            this.worldCharts.add(chartId);
+        }
+        if (!this.worldCharts.has(chartId)) {
+            return [];
+        }
+        const geoJsonService = this.custom.env.services.geo_json_service;
+        return (geoJsonService?.getAvailableRegions() || []).filter(
+            (region) => region.id !== "usa"
+        );
```

Comment:
That kinda defeats the whole purpose of having a generic geoJsonService doesn't it ? >_<

IMO there should be a `getZoomed/AlternativeRegions` method in the geoJsonService, and this plugin should remember what the original chart region was